src/ApiRequestPanel.js

```diff
@@ -517,6 +543,9 @@ export class ApiRequestPanel extends EventsTargetMixin(HeadersParserMixin(LitEle
     this._appendConsoleHeaders(e);
     this._appendProxy(e);
   }
+  _serverChangeHandler(e) {
+    this.selectedServer = e.detail.value;
```

Comment:
So here you would set a variable that sets an information that the base URI is set from the selector and the selector should continue to be rendered. Then you need to clear it when the selection change.

Reply:
Not needed, see my comment above